### ISSUE TITLE
UVR doesn't work with screenreaders

### ISSUE THREAD

**Issue by VIPPotato** [Anjok07/ultimatevocalremovergui#418]
So  I'm interested in checking out the ultimate vocal remover, but the GUI is totally inaccessible for screenreaders. I'm using non visual desktop access. Sadly nothing is being read in the app's window.
